Lib9c.Policy/NCStagePolicy.cs

```diff
@@ -59,7 +59,17 @@ public IEnumerable<Transaction> Iterate(BlockChain blockChain, bool filtered = t
                     }
 
                     s.Add(tx);
-                    if (s.Count > _quotaPerSigner)
+                    int txQuotaPerSigner = _quotaPerSigner;
+                    if (_accessControlService != null)
+                    {
+                        // update txQuotaPerSigner if ACS returns a value for the signer.
+                        if (_accessControlService.GetTxQuota(tx.Signer) is { } acsTxQuota)
+                        {
+                            txQuotaPerSigner = acsTxQuota;
+                        }
+                    }
```

Comment:
It uses the `IAccessControlService.GetTxQuota` method only. I think it is simple to use *null propagation*.

```suggestion
                    // update txQuotaPerSigner if ACS returns a value for the signer.
                    if (_accessControlService?.GetTxQuota(tx.Signer) is { } acsTxQuota)
                    {
                        txQuotaPerSigner = acsTxQuota;
                    }
```